config/locales/en.yml

```diff
@@ -1658,6 +1658,79 @@ en:
             allow_registration_without:
               "true": "Competitors will be allowed to register for events they're not qualified for (I'll remove them later)"
               "false": "Competitors MUST be qualified to register for an event"
+        #context: new registration system
+    registration_v2:
+      pdf:
+        #context: <0> indicates an html tag for internationalizing in React, see https://docs.worldcubeassociation.org/guides/translating_the_website.html#using-react
+        link: "As a <0>%{pdfLink}</0>"
+        download: "Download all of the competitions details."
+      list:
+        empty: "No matching Records found"
+        spots_remaining: "; %{spots} spot(s) remaining"
+        waitlisted: "Waitlisted registrations"
+        comment: "Comment"
+        admin_note: "Admin Note"
+      tabs:
+        payment: "Payment"
+        more: "More"
+      fees:
+        none: "No Entry Fee"
+        guest_none: "Guests attend for free"
+      bookmark:
+        unbookmark: "You have removed your bookmark"
+      requirements:
+        title: "Requirements"
+        acknowledgement: "I have read and acknowledged all information under the 'Register' tab, including entry fees and refund policies"
+        acknowledgment_extra: "I have read and understood all information listed above"
+        next_step: "Continue to next Step"
+      update:
+        no_changes: "There are no changes"
+        email_send: "Send Email"
+        email_copy: "Copy Email"
+        move_pending: "Move to Pending"
+        move_waiting: "Move to Waiting List"
+        cancel: "Cancel Registration"
+        too_many: "Accepting all these registrations would go over the competitor limit by %{count}"
+      register:
+        error: "Registration failed with error %{error}"
+        early_registration: "Registration is not open yet, but you can still register as a competition organizer or delegate."
+        comment: "Additional comments to the organizers"
+        until: "You can update your registration until %{date}"
+        passed: "You can no longer update your registration"
+        editing_disabled: "Registration editing is disabled for this competition"
+        registration_status:
+          header: "Your Registration Status:"
+          accepted: "accepted"
+          pending: "pending"
+          waiting: "Waiting List"
+          cancelled: "cancelled"
```

Comment:
Inconsistent capitalization. Also, we probably already have those.

Reply:
There is 
```
competition_medium:
      status:
        accepted: "Accepted"
        pending: "Pending"
```
But what I want to do is just `t(competitions.registration_v2.register.registrationStatus.${registration.competing.registration_status})`, but I can't do that if half of the statusses are in one key and the other in the other. A solution would be to add those to `competition_medium.status,` but that would cause and unused key error

Reply:
In that case, move the other new status flags to the same place where the old ones already are and add ignore rules to the `i18n-tasks.yml` config to suppress the unused keys